numpy/core/src/umath/ufunc_object.c

```diff
@@ -5888,6 +5888,13 @@ ufunc_at(PyUFuncObject *ufunc, PyObject *args)
 
     NPY_BEGIN_THREADS_DEF;
 
+    if (ufunc->core_enabled) {
+        PyErr_Format(PyExc_TypeError,
+            "numpy.%s.at does not support ufunc with non-trivial signature: %s has signature %s.",
```

Comment:
```suggestion
            "%s.at does not support ufunc with non-trivial signature: %s has signature %s.",
```
Ufuncs don't necessarily live in NumPy, and functions errors don't usually include the module anyway luckily.